### PR TITLE
task(rename): move project to maistra org

### PR DESCRIPTION
**Pull request by aslakknutsen**
related to #84

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -111,7 +111,7 @@ DOCKER_IMAGE?=$(PROJECT_NAME)
 DOCKER_IMAGE_TAG?=$(COMMIT)
 export DOCKER_IMAGE_TAG
 DOCKER_REGISTRY?=docker.io
-DOCKER_REPOSITORY?=aslakknutsen
+DOCKER_REPOSITORY?=maistra
```
Comment (bartoszmajsak) [inline Makefile]: @aslakknutsen I guess this we wanted to keep in the old account before we sort out access to docker hub?

Comment (aslakknutsen) [reply Makefile]: ah right!

File: pkg/apis/istio/v1alpha1/zz_generated.deepcopy.go [hunk 2]
```diff
@@ -1,5 +1,21 @@
 // +build !ignore_autogenerated
 
+/*
+Copyright The Kubernetes Authors.
```
Comment (bartoszmajsak) [inline pkg/apis/istio/v1alpha1/zz_generated.deepcopy.go]: Why do we see this all of the sudden?

Comment (aslakknutsen) [reply pkg/apis/istio/v1alpha1/zz_generated.deepcopy.go]: hmm.. Good question. Some lib update I guess

File: Makefile (hunk 1 above)
Comment (aslakknutsen) [reply Makefile]: https://github.com/Maistra/istio-workspace/pull/111/commits/7c97fbf255f0e77e123360580ce257d2de71234e

File: README.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
-image:https://goreportcard.com/badge/github.com/aslakknutsen/istio-workspace["Go Report Card", link="https://goreportcard.com/report/github.com/aslakknutsen/istio-workspace"]
-image:https://circleci.com/gh/aslakknutsen/istio-workspace.svg?style=svg["CircleCI", link="https://circleci.com/gh/aslakknutsen/istio-workspace"]
+image:https://goreportcard.com/badge/github.com/maistra/istio-workspace["Go Report Card", link="https://goreportcard.com/report/github.com/maistra/istio-workspace"]
+image:https://circleci.com/gh/maistra/istio-workspace.svg?style=svg["CircleCI", link="https://circleci.com/gh/maistra/istio-workspace"]
```
Comment (bartoszmajsak) [inline README.adoc]:
```suggestion
image:https://circleci.com/gh/Maistra/istio-workspace.svg?style=svg["CircleCI", link="https://circleci.com/gh/Maistra/istio-workspace"]
```

**Pull request merged**